Makefile

```diff
@@ -10,7 +10,9 @@ export GOPATH := $(GOPATH):$(CURDIR)/_vendor
 all: go rust test
 
 test:
-	cd pkg/util && go test
+	# Build test.
+	go build ./pkg/...
+	cargo build
```

Comment:
I think `cargo check` is enough. It is available since 1.16.0 stable release(Mar 16, 2017). https://blog.rust-lang.org/2017/03/16/Rust-1.16.html